### ISSUE TITLE
installation (instructions)

### ISSUE THREAD

**Issue by orlin** [dmitryame/echowaves#2]
- sudo gem install ruby-debug
- rake gems:install

error: "Could not find RubyGem oauth (>= 0.2.1)"
rake doesn't install neither pelle-oauth (0.3.1) nor oauth (0.3.2)
recognizes existing / does nothing after: "sudo gem install oauth"
- sqlite3: "Migration error: Unsupported database for migration to UTF-8 support"
- osx: fink's apt-get can't find twisted package (use dmg from twistedmatrix.com)
- sudo cp config/orbited.cfg /etc/
- this rake task http://bit.ly/JBwHm can be tweaked for our purposes

in conclusion, we could have a rails tamplate to rake into a cloned echowaves

**Comment by stewiebigz-art** [comment]
<img width="1024" height="768" alt="Image" src="https://github.com/user-attachments/assets/1bf26701-cc81-470e-9b49-c5c26a8be404" />
